resources/views/layouts/base.blade.php

```diff
@@ -87,6 +87,17 @@ class="container mx-auto px-4 mt-4 md:mt-12 max-w-[1200px] mb-8"
     {{ $slot }}
 </div>
 
+@if(request()->is('rfc/*'))
```

Comment:
I don't think this is the way to go. The rfc view should pass a property to the base component like we already do with the page title:

```blade
@component('layouts.base', [
    'pageTitle' => $rfc->title . ' – RFC Vote',
])
```